nextcloud/server
# Bump nextcloud-dialogs from 3.1.2 to 3.1.4

This brings in the "aria-live" attribute for accessibility.

To test:

1. Create a folder "abc"
2. Open a second browser window on the root
3. In the second window, rename "abc" to "def"
4. Go back to the first browser window and also rename "abc" to "def" to trigger an error
5. Inspect the notification

<img width="789" alt="image" src="https://user-images.githubusercontent.com/277525/180239309-3f23c61a-c401-4004-ac90-0957b5af4020.png">

We can see that aria-live is there.

**BUT** there's also this weird button size now, perhaps due to my increased font sizes.
Not sure, but could be caused by https://github.com/nextcloud/nextcloud-dialogs/pull/582 as it touches the button
